### ISSUE TITLE
Don’t combine words when merging sequential links

### ISSUE THREAD

**Issue by DanaDangerGrey** [instructure/tinymce-a11y-checker#11]
They A11y Checker can return an issue of "Adjacent links with the same URL should be a single link." There is a checkbox you can click that will automatically merge the links. Currently, when you check this, it removes the space between the two words. 
Example: "ClickHere" instead of "Click Here"

**To Reproduce**
Give two adjacent words the same link, but independently (link each word separately). Run A11y Checker. Check the box to Merge the links.

**Expected behavior**
It should not remove spaces between words when combining the links.
